apps/web/src/pages/settings/organization/OrganizationLogo.tsx

```diff
@@ -0,0 +1,65 @@
+import { errorMessage } from '@novu/design-system';
+import { MimeTypesEnum } from '@novu/shared';
+import React, { useEffect } from 'react';
+import { Controller, useForm } from 'react-hook-form';
+import { ProfileImage } from '../../../components/shared';
+
+type FormValues = {
+  logoUrl: string;
+};
+
+const MIME_TYPES = {
+  'image/jpeg': MimeTypesEnum.JPEG,
+  'image/png': MimeTypesEnum.PNG,
+};
+
+export function OrganizationLogo({ logoUrl }: { logoUrl?: string }) {
+  const { control, reset } = useForm<FormValues>({
+    defaultValues: {
+      logoUrl,
+    },
+  });
+
+  /**
+   * Reset the form when the organization changes
+   * Useful in the scenario when the user switches to a different organization
+   */
+  useEffect(() => {
+    reset({ logoUrl });
+  }, [logoUrl, reset]);
+
+  async function handleUpload(event: React.ChangeEvent<HTMLInputElement>) {
+    const file = event.target.files?.[0] ? event.target.files[0] : null;
+
+    if (!file) {
+      return;
+    }
+
+    const fileExtension = file.type;
+    const isValidFileType = MIME_TYPES[fileExtension] !== undefined;
+    if (!isValidFileType) {
+      errorMessage('Invalid file type. Please upload a PNG or JPEG file.');
+
+      return;
+    }
+    //TODO: Add organization logo api implementation once it is available
```

Comment:
```suggestion
    // FIXME: Add organization logo api implementation once it is available
```
‏
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
✏ **suggestion:** We can discuss this as a general discussion for eng, but I suggest using `FIXME‏` for things that shouldn't be merged to `next` without being done, and `TODO` for general action items that may be less immediate. It makes things easier to search, and we could even implement a CI rule that catches FIXMEs in changed files